src/TodoWrapper/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Todos } from '../Todos';
+import { TodosForm } from '../TodosForm';
+import { Filtering } from '../Filtering';
+import { useParams } from 'react-router-dom';
+import { Paper, Box } from '@material-ui/core';
+
+export const TodoWrapper = () => {
+  const { userId } = useParams();
+  const [showAll, setShowAll] = React.useState(false);
+  const [todos, setTodos] = React.useState([]);
+
+  React.useEffect(() => {
+    const fetchUser = async () => {
```

Comment:
please change all variables from user to todos, such as `fetchUser` to `fetchTodos`